services/brig/docs/swagger-v7.json

```diff
@@ -1998,6 +1763,206 @@
         ],
         "type": "object"
       },
+      "CreateUserTeam": {
+        "properties": {
+          "team_id": {
+            "$ref": "#/components/schemas/UUID"
+          },
+          "team_name": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "team_id",
+          "team_name"
+        ],
+        "type": "object"
+      },
+      "Currency.Alpha": {
```

Comment:
why is it Currency.Alpha now and was Alpha before?

did the version of any relevant dependency change?  or did the namespace change so that this needs to be better qualified now?  (is qualifying even a thing here?)

and why did this happen now, and now in https://github.com/wireapp/wire-server/pull/4356 ?